java/ql/src/experimental/Security/CWE/CWE-094/JShellInjection.ql

```diff
@@ -21,6 +21,19 @@ class JShellInjectionConfiguration extends TaintTracking::Configuration {
   override predicate isSource(DataFlow::Node source) { source instanceof RemoteFlowSource }
 
   override predicate isSink(DataFlow::Node sink) { sink instanceof JShellInjectionSink }
+
+  override predicate isAdditionalTaintStep(DataFlow::Node prod, DataFlow::Node succ) {
```

Comment:
I assume this is a typo?
It should probably be **`pred`** ecessor instead of `prod`.

Reply:
You're right.